Makefile

```diff
@@ -25,16 +29,50 @@ help:
 	@echo
 	@grep -E '^### [-A-Za-z0-9_]+:' Makefile | sed 's/###/   /'
 
-export GO111MODULE=on
 
-### license-check:    Check apisix-dashboard source codes for Apache License
-.PHONY: license-check
-license-check:
-ifeq ("$(wildcard .actions/openwhisk-utilities/scancode/scanCode.py)", "")
-	git clone https://github.com/apache/openwhisk-utilities.git .actions/openwhisk-utilities
-	cp .actions/ASF* .actions/openwhisk-utilities/scancode/
+### dashboard-build:   build dashboard, it contains frontend and manager-api
```

Comment:
`it contains frontend and manager-api`
to 
`it contains web and manager-api`

keep same as the folder name. @juzhiyuan what do you think?

Reply:
Yep